class/practice2.1/testCases.txt

```diff
@@ -1,123 +1,133 @@
-Test case 1
+Test case "Normal usage"
 Precondition: none
```

Comment:
There should be preconditions as we already have database file that impacts program behavior. Therefore to make test case reproducible we should specify state of the file. For example the very first run is en equivalent to remove the file as precondition. also you can emulate some scenarios just by set proper data into that file before the test.